### ISSUE TITLE
Adopt new toolbar UI

### ISSUE THREAD

**Issue by christoph-kluge** [fecoderchinh/gallery-editorjs#2]
I guess the current styles apply to the <2.23. toolbar style. With the new toolbar (left-side of the content) it seems a bit odd.

<img width="906" alt="Screenshot 2022-06-07 at 13 43 50" src="https://user-images.githubusercontent.com/1446269/172371677-e01fc1f9-2586-4800-bfc1-3d82b2c8e9c7.png">

**Comment by fecoderchinh** [comment]
Yeah, i knew that's bit silly and oddly, but at this moment, i still don't have any idea for the design.
Also, these icons were made to custom the number of column if needed.
At the last time, i wanted to put them inside the main settings, but it broke the toggle's logic.
